tests/test_python.py

```diff
@@ -88,6 +89,20 @@ def test_python_matrix(
     if python_version == "3.11" and profiler_type == "pyperf":
         pytest.xfail("PyPerf does not support 3.11 - https://github.com/Granulate/gprofiler/issues/727")
 
+    if is_aarch64():
+        # pyperf is not working on aarch right now https://github.com/Granulate/gprofiler/issues/499
+        if profiler_type == "pyperf":
+            pytest.skip("PyPerf doesn't support aarch64 architecture!")
+
+        if python_version == "2.7" and profiler_type == "py-spy" and app == "uwsgi":
+            pytest.xfail("This combination fails, see https://github.com/Granulate/gprofiler/issues/713")
+
+        if python_version == "3.11" and profiler_type == "py-spy" and libc == "glibc":
+            pytest.xfail("This combination fails, see https://github.com/Granulate/gprofiler/issues/740")
```

Comment:
See my comment https://github.com/Granulate/gprofiler/issues/740#issuecomment-1481294047 - this might work after you merge master. Please try and remove if it now fails.